### PR TITLE
disable context menu

### PR DESCRIPTION
**Pull request by yteslenk**

### CODE REVIEW

File: packages/wix-ui-core/src/components/video/players/Playable.tsx [hunk 1]
```diff
@@ -175,6 +175,7 @@ class PlayablePlayer extends React.PureComponent<
       hideOverlay: true,
     });
 
+    this.player.getElement().addEventListener('contextmenu', event => event.preventDefault());
```
Comment (kupriyanenko) [inline packages/wix-ui-core/src/components/video/players/Playable.tsx]: 1. you can use `handleRightClick` here.
2. you need to remove listener on unmount

**Comment by yteslenk** [comment]
@shlomitc, please merge it)

**Comment by yteslenk** [comment]
$AUTO_MERGE$

**Comment by kupriyanenko** [comment]
@shlomitc can you please merge it

**Comment by mokhovyk** [comment]
@piratus could you merge?

**Pull request merged**